### ISSUE TITLE
Colors and characters are missing

### ISSUE THREAD

**Issue by enko** [FormidableLabs/nodejs-dashboard#15]
Have a look at these two screenshots:

![bildschirmfoto_2016-10-13_09-25-58](https://cloud.githubusercontent.com/assets/19407/19340566/d6012e70-9129-11e6-89d6-44085da9b825.png)

vs

![bildschirmfoto_2016-10-13_09-26-35](https://cloud.githubusercontent.com/assets/19407/19340570/db974806-9129-11e6-9d0f-216b004f5e94.png)

It seems the ansicodes and unicode characters of mocha are beeing striped?

Check this gist for reproduction:

https://gist.github.com/enko/31be945ca47b0c481275e76a2b695714

Run `docker-compose build` and then `docker-compose run test_runner` alternativly append `-f docker-compose_colors.yml` to run it without nodejs-dashboard.

This also repros #10.

Thanks for your time for this nice product!

**Comment by jasonwilson** [comment]
Pretty sure this is caused by `isTTY` failing in the child process since the parent isn't passing through stdio. As a quick test I updated your mocha exec command to force colors `nodejs-dashboard -- mocha test.js --colors` and it reproduces them correctly.

The trick here is finding a way to force tty with a child process. The issue is detailed nicely here: https://github.com/nodejs/node-v0.x-archive/issues/2754

**Comment by ryan-roemer** [comment]
For `chalk`-based apps / scripts, the `FORCE_COLOR="true"` env variable will force colors to be enabled. Not sure if it's appropriate for the dashboard itself to hack that in though... Possibly could have a dashboard `--colors` flag to "do all hacks possible to do colors that we know of".

See https://github.com/chalk/supports-color/blob/master/readme.md#info

**Comment by jjasonclark** [comment]
I can confirm that adding the environment values will fix both issues.

Add the following values to your `docker-compose.yml` file
```
    environment:
      LANG: en_US.UTF-8
      FORCE_COLOR: 'true'
```

I forked @enko 's gist to include adding the environment values. The tests run with the graphs showing the correct dots and the color in the stdout window. https://gist.github.com/jjasonclark/0afed89466db871a3479f55ca2534439

`docker-compose run --rm test_runner`

![issue_15](https://user-images.githubusercontent.com/155233/33355300-c6013ca0-d46b-11e7-983c-e704510b7fbc.png)

Closing as complete
